frontend/packages/integration-tests-cypress/views/logs.ts

```diff
@@ -0,0 +1,33 @@
+export const logs = {
+  toggleOptions: () => {
+    // click configuration dropdown
+    cy.get('button[data-test="resource-log-options-toggle"]').click();
+  },
+  checkLogWraped: (flag: boolean) => {
+    // open options
+    logs.toggleOptions();
+    if (flag) {
+      cy.get('li[data-test-dropdown-menu="wrap-lines"] input').should('be.checked');
+    } else {
+      cy.get('li[data-test-dropdown-menu="wrap-lines"] input').should('not.be.checked');
+    }
+    // close options
+    logs.toggleOptions();
+  },
+  selectContainer: (containerName: string) => {
+    cy.get('button[data-test="container-select"]').click();
+    cy.get(`li[data-test-item-name=${containerName}] button`).click();
```

Comment:
updated selector

Reply:
we can probably just use the `byTestDropdownMenu` / `byTestId` here since we know the test ids are unique